OpenUpSA/wazimap-ng-ui
# Only use filter attributes that exist in the chart dataset

## Description

If the configuration tries to use a filter attribute that does not exist in the dataset, find and use one that does

## Related Issue
https://tree.taiga.io/project/jbothma-wazimap-ng/task/634
https://trello.com/c/5kiAboxy/824-default-filter-option-is-displayed-despite-it-not-being-a-valid-option

## How to test it locally
On default YE profile navigate to data mapper and select Education > Progress through school > Grade 9> No 
Check whether the filter attribute is correct or if it's used some unknown filter attribute that is not shown

## Screenshots


## Changelog

### Added

### Updated
Setting non-existent default filters in admin chart configuration will no longer silently break the front end
### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
